### DISCUSSION TITLE
의자에 표시된 Press X 글자가 잘 안보임

### DISCUSSION THREAD

**Discussion by neopandora** [metamong-friends/metamong-friends.github.io#10] [제보하기]
의자에 표시된 Press X 글자가 흰색이어서 연한 색상의 의자에서는 글자가 잘 보이지 않음

**Comment by ACT-TJ** [comment]
유저의 위치나 각도, 가구 색상 등에 따라 해당 플로팅 문구의 가시성이 떨어지는 것으로 
보이네요. 변경에 대해서 검토해 보겠습니다.! 의견 감사드립니다!
